src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -262,13 +262,19 @@
   .method public hidebysig static !!T& AsRef<T>(void* source) cil managed aggressiveinlining
   {
         .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+#ifdef NETCORE20
+        .maxstack 1
+        ldarg.0
+        ret
+#else
```

Comment:
@karelz added comment. Waiting for feedback regarding packaging before this can be completed.